etc/picongpu/pizdaint-cscs/picongpu.profile.example

```diff
@@ -25,19 +25,29 @@ else
     module load PrgEnv-gnu/6.0.4
 fi
 
-module load cudatoolkit/8.0.61_2.4.3-6.0.4.0_3.1__gb475d12
+module load daint-gpu
+# currently loads CUDA 8.0
+module load craype-accel-nvidia60
+
+# Compile for cluster nodes
+#   (CMake likes to unwrap the Cray wrappers)
+export CC=$(which cc)
+export CXX=$(which CC)
```

Comment:
Question:
Both executables are identical. Is this okay? (They point both to the executable `driver`.)

Reply:
because they are controlled further by env vars.